annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastCreator.java

```diff
@@ -0,0 +1,49 @@
+package com.exactpro.epfast.annotation.processing;
+
+import javax.lang.model.element.Element;
+import java.util.*;
+
+public class FastCreator {
+
+    private ArrayList<AbstractFastPackage> packageElements = new ArrayList<>();
+
+    private AnonymousFastPackage rootPackage;
+
+    public AbstractFastPackage getFastPackage(String fastPackageName) {
+        return packageElements.stream()
+            .filter(packageElement -> packageElement.getName().equals(fastPackageName))
+            .findFirst().orElse(null);
+    }
+
+    public Collection<AbstractFastPackage> getAllFastPackages() {
+        return packageElements;
+    }
+
+    public AbstractFastPackage getRootPackage() {
+        return rootPackage;
+    }
+
+    public void removePackage(String packageName) {
+        packageElements.removeIf(packageElement -> packageElement.getName().equals(packageName));
+    }
+
+    public void add(Element fastType, String fastPackageName) {
```

Comment:
change it to `add(Element fastType)` or add(FastTypeElement fastType)`. Place logic which uses resolver to this method